packages/benefit-price/src/BenefitPrice.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Price from '@hixme-ui/price'
+
+const BenefitPrice = ({
+  noLabel,
+  payCycle,
+  value,
+  ...restProps
+}) => {
+  const getLabel = () => {
```

Comment:
We should move getLabel and getPrice outside of the component and let them take an argument(s), and build tests around them.